### ISSUE TITLE
1.19.3 support

### ISSUE THREAD

**Issue by ZomkaDEV** [Bloom-host/Petal#31]
Petal is great but I really hope that 1.19.3 support would be added soon! :)

**Comment by Kadeluxe** [comment]
It's not great, they failed

**Comment by kaxonomy** [comment]
> It's not great, they failed

How

**Comment by Kadeluxe** [comment]
> > It's not great, they failed
> 
> How

Zero support, subtle performance gains while introducing AI issues, the whole thing was overhyped. No point to use petal currently.

**Comment by TH3N3WN00B** [comment]
> > > It's not great, they failed
> > 
> > 
> > How
> 
> Zero support, subtle performance gains while introducing AI issues, the whole thing was overhyped. No point to use petal currently.

They always said never to use it in prod, and that it was still in beta and under development, as well as dev has other job and he will code during the weekends, give it some time to fix all the issues, meanwhile use purpur

**Comment by Kadeluxe** [comment]
Yes, that's why there is no point in using petal and imo I think it's a dead project, given their "dev" was working alone.
